sample_apps/dotnet/QueryExample.cs

```diff
@@ -278,7 +279,7 @@ private void ParseQueryResult(QueryResponse response)
             List<ColumnInfo> columnInfo = response.ColumnInfo;
             var options = new JsonSerializerOptions
             {
-                IgnoreNullValues = true
+                DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull
```

Comment:
Since it is a sample app - add code documentation inline explaining what this property does.